pages/admin/dashboard/tools.tsx

```diff
@@ -27,11 +27,21 @@ const ToolsPage: React.FC = () => {
     // TODO: Need to update this to calculate relative date
     const date = new Date();
 
-    const filterTools = () => {
-        const filteredTools = toolsList.filter((t) => {
-            return t["status"] === selectedTab;
-        });
-        setToolsArray(filteredTools);
+    const filterTools = async () => {
+        try {
+            const response = await axios({
+                method: "GET",
+                url: "/api/tool/getAll",
+            });
+
+            const filteredTools = response.data.filter((t) => {
+                return t["status"] === selectedTab;
+            });
+
+            setToolsArray(filteredTools);
+        } catch (err) {
+            console.log("error");
```

Comment:
Not sure if logging to console is the best idea but if we do we should also log the actual error to the console

Reply:
I think we don't have any error handling in place yet, will log the actual error to the console and keep a todo comment for handling the error